src/main/msp/msp.c

```diff
@@ -3504,6 +3516,15 @@ static mspResult_e mspProcessInCommand(mspDescriptor_t srcDesc, int16_t cmdMSP,
 #else
             sbufReadU8(src);
 #endif
+        }
+        if (sbufBytesRemaining(src) >= 1) {
+            // Added in MSP API 1.44
```

Comment:
should be 1.45

Reply:
Can you verify 1.45 is correct? As the code stands, the MSP_API_VERSION command will return 1.44. I had originally changed that to 1.45, but I learned that's something the admins do, so I reverted that change. I want to make sure my matching change on the configurator is backward compatible with existing firmwares

Reply:
I suspect my other comment referencing 1.45 is the incorrect one

Reply:
As long as this PR is on the 4.4 milestone it should be 1.45. The version will be incremented once 4.3.0 is released and work on 4.4 starts.